try{
    raise NameError("A")
}
except NameError t{
    print(t)
}
finally{
    print("done")
}
